docs/book/features/pv_to_backingdiskobjectid_mapping.md

```diff
@@ -0,0 +1,92 @@
+  
+  ```bash
+  $ kubectl patch configmap/internal-feature-states.csi.vsphere.vmware.com \
+  -n vmware-system-csi \
+  --type merge \
+  -p '{"data":{"pv-to-backingdiskobjectid-mapping":"true"}}'
+  ```
+  
+  Verify that `pv-to-backingdiskobjectid-mapping` is enabled:
+  
+  ```bash
+  # kubectl edit configmap internal-feature-states.csi.vsphere.vmware.com -n vmware-system-csi
+  apiVersion: v1
+  data:
+    ...
+    pv-to-backingdiskobjectid-mapping: "true"
+    ...
+  ```
+  
+## How to use PV to BackingDiskObjectId Mapping feature <a id="how-to-use"></a>
+
+- Here is an example of the vsphere-csi-driver deployment:
+
+  ```bash
+    $ kubectl -n vmware-system-csi get pod,deploy
+    NAME                                          READY   STATUS    RESTARTS   AGE
+    pod/vsphere-csi-controller-5547b8bd54-bdtwz   6/6     Running   0          6h40m
+    pod/vsphere-csi-node-l4zmm                    3/3     Running   0          6h44m
+    pod/vsphere-csi-node-m2l7x                    3/3     Running   0          6h44m
+    pod/vsphere-csi-node-phdqk                    3/3     Running   0          6h44m
+    pod/vsphere-csi-node-r7r2v                    3/3     Running   0          6h44m
+    pod/vsphere-csi-node-rzk9q                    3/3     Running   0          6h44m
+    pod/vsphere-csi-node-sg56b                    3/3     Running   0          6h44m
+
+    NAME                                     READY   UP-TO-DATE   AVAILABLE   AGE
+    deployment.apps/vsphere-csi-controller   1/1     1            1           15h
+  ```
+
+- After `pv-to-backingdiskobjectid-mapping` feature is enabled, restart vsphere-csi-controller pod.
+
+  ```bash
+  kubectl -n vmware-system-csi scale deploy/vsphere-csi-controller --replicas=0
+  kubectl -n vmware-system-csi scale deploy/vsphere-csi-controller --replicas=1
+  ```  
+
+- Check PVC annotation to find the pv to backingdiskobjectid mapping info:
+
+  ```yaml
+  root@k8-master-657:~# kubectl get pvc -n test-ns-swmiznp -o yaml
+  apiVersion: v1
+  items:
+  - apiVersion: v1
+    kind: PersistentVolumeClaim
+    metadata:
+      annotations:
+        cns.vmware.com/pv-to-backingdiskobjectid-mapping: 61ffa393-b5da-4963-a4c0-7ead9b3696c0:6795fc61-3458-fe61-5e85-020078562642
+        kubectl.kubernetes.io/last-applied-configuration: |
+          {"apiVersion":"v1","kind":"PersistentVolumeClaim","metadata":{"annotations":{},"name":"etcd0-pv-claim","namespace":"test-ns-swmiznp"},"spec":{"accessModes":["ReadWriteOnce"],"resources":{"requests":{"storage":"1Gi"}},"storageClassName":"kibishii-storage-class"}}
+        pv.kubernetes.io/bind-completed: "yes"
+        pv.kubernetes.io/bound-by-controller: "yes"
+        volume.beta.kubernetes.io/storage-provisioner: csi.vsphere.vmware.com
+      creationTimestamp: "2022-02-04T02:54:07Z"
+  ```
+  
+  For this example pv which is provisioned on vSAN datastore:
+  PV UID: 61ffa393-b5da-4963-a4c0-7ead9b3696c0
+  BackingDiskObjectId: 6795fc61-3458-fe61-5e85-020078562642
```

Comment:
This formatting does not look right when using "View File". 
Add "- " in front of line 91 and 92 so they become bullet points.